src/test/java/no/unit/nva/model/contexttypes/UnconfirmedSeriesTest.java

```diff
@@ -0,0 +1,44 @@
+package no.unit.nva.model.contexttypes;
+
+import static no.unit.nva.model.util.PublicationGenerator.randomInvalidIssn;
+import static no.unit.nva.model.util.PublicationGenerator.randomIssn;
+import static no.unit.nva.utils.RandomData.randomString;
+import static nva.commons.core.JsonUtils.objectMapperWithEmpty;
+import static nva.commons.core.attempt.Try.attempt;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import no.unit.nva.model.exceptions.InvalidIssnException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+class UnconfirmedSeriesTest {
```

Comment:
Does this cover something not covered in more generic tests elsewhere?

Reply:
We do not cover the cases where the issns are invalid

Reply:
I checked and we do not have test coverage reports for the datamodel-java

Reply:
I have added a test for deserialzation of unconfirmed series.  I think trying to inject an invalid JSON UnconfirmedSeries object to a Publication JSON object, would require a lot of code and syncing between fields. In addition given the many objects that a Publication has and the many cases per object that must be tested, this could lead to an large number of tests and a need for complex data generation.

Example:
It is easier to test the de/serialization features of UnconfirmedSeries focusing only on this object and having tests such as 
"deserialization suceeds when input contains valid issn and valid online issn" 
"deserialization fails when input contains invalid issn"
than to test
"deserialization succeeds when a Publication refers to Book with Unconfirmed Series  that contains a valid issn and valid online issn"
"deserialization fails when a Publication refers to Book with Unconfirmed Series  that contains an valid issn."